pkg/config/config.go

```diff
@@ -1537,6 +1537,14 @@ func (r *RuntimeHandler) ValidateRuntimePath(name string) error {
 			name)
 	}
 
+	versionOutput, err := cmdrunner.CombinedOutput(r.RuntimePath, "--version")
+	if err == nil {
+		versionString := strings.ReplaceAll(strings.TrimSpace(string(versionOutput)), "\n", ", ")
```

Comment:
crun's output will look a bit oof here. :smile:

Reply:
Hm, I assume that's okay:
```
Using runtime crun version 1.14.4, commit: 1.14.4, rundir: /run/crun, spec: 1.0.0, +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +YAJL
```

Reply:
Not so bad, actually. I was worried about the build flags...